Remove irrelevant information about IE (#36281)
mdn/content: files/en-us/web/accessibility/keyboard-navigable_javascript_widgets/index.md

> d---

Reviewer: ```suggestion
---
```

### Always draw the focus for tabindex="-1" items and elements that receive focus programmatically

> Ensure that focused elements have the focus ring. This can be done using the CSS {{cssxref("outline")}} property. Avoid unconditionally setting it to `none`—if you want to prevent unnecessary focus rings being displayed, use the {{cssxref(":focus-visible")}} pseudo-class.

Reviewer: ```suggestion
Ensure that focused elements have a focus ring. This can be done using the CSS {{cssxref("outline")}} property, which should not be unconditionally set to `none`—if you want to prevent unnecessary focus rings being displayed, use the {{cssxref(":focus-visible")}} pseudo-class.
```
